src/main/java/Task.java

```diff
@@ -0,0 +1,60 @@
+public abstract class Task {
```

Comment:
Should all files be in packages?

Reply:
Thanks! I agree that all classes should belong to packages so I've since made that change